jobs/webgroup/blockbot.hcl

```diff
@@ -0,0 +1,32 @@
+job "blockbot" {
+  datacenters = ["aperture"]
+
+  type = "service"
+
+  group "blockbot" {
+    count = 1
+
+    task "blockbot" {
+      driver = "docker"
+
+      config {
+        image = "ghcr.io/redbrick/blockbot"
+        entrypoint = ["python3", "src/main.py"]
```

Comment:
This should really be defined in the Dockerfile

Reply:
It is, so all good 👍 I'll remove the line